### DISCUSSION TITLE
EventToCommandBehavior should pass arguments by default without needing to specify TypeArguments

### DISCUSSION THREAD

**Discussion by symbiogenesis** [CommunityToolkit/Maui#917] [New Feature Discussions]
[This tip](https://github.com/CommunityToolkit/Maui/issues/569#issuecomment-1306170318) about x:TypeArguments turned out to be very important for me, but since every implementation of EventToCommandBehavior that I had used in the past didn't need this kind of help, it seems like the CommunityToolkit should not require it to pass parameters.

Wouldn't it be better if the default behavior used reflection to discover the argument type?

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
